zerver/openapi/zulip.yaml

```diff
@@ -591,7 +591,7 @@ paths:
           **Changes**: Support for using user/stream IDs was added in Zulip 2.0.0.
         schema:
           type: string
-        example: [9, 10]
+        example: "[9, 10]"
```

Comment:
I think this has the same issue as the others -- we actually want to be recording an `application/json` list of integers here, not a "string"...

Reply:
Fixed.